Feature/angular boilerplate

CloudNiner opened the pull request.

CloudNiner committed:
Remove old static placeholder files

CloudNiner committed:
Add dev port for grunt serve

CloudNiner committed:
Vagrant nginx serves webapp dist folder

CloudNiner committed:
Add boilerplate angular app

Remove unnecessary htaccess file

CloudNiner committed:
Add blank nih.profiles module

CloudNiner committed:
Add angular ui-router and bootstrap js directives

Add bower deps to karma conf

CloudNiner committed:
Set configs to 4 spaces

CloudNiner commented:
This is still very boilerplate, but it should be enough to unblock @flibbertigibbet. I added a blank nih.profiles module.

Things I'm still going to do as part of this card:
- Set up the views defined in the wireframes using ui-router
- ngARIA
- Tweak jshint settings
- Install compass, bower, grunt-cli as part of the ansible provisioning process

kshepard commented on src/nih_wayfinding/Gruntfile.js:
```diff
@@ -0,0 +1,444 @@
+    connect: {
+      options: {
+        port: 9000,
+        // Change this to '0.0.0.0' to access the server from outside.
```
Update this comment

kshepard commented on src/nih_wayfinding/Gruntfile.js:
```diff
@@ -0,0 +1,444 @@
+      }
+    },
+
+    // The following *-min tasks will produce minified files in the dist folder
```
Any reason these are commented out?

CloudNiner replied:
That's part of the generated boilerplate, it shouldn't apply to us. Going to remove it.

kshepard commented:
I'm assuming that's the Yeoman favicon. Let's ditch it before it gets cached in our browsers.

kshepard commented on src/nih_wayfinding/test/.jshintrc:
```diff
@@ -0,0 +1,36 @@
+  "curly": true,
+  "eqeqeq": true,
+  "immed": true,
+  "indent": 2,
```
Should be a 4

CloudNiner replied:
crap. missed one.

kshepard commented:
+1

CloudNiner committed:
Misc PR cleanup

  - Remove yeoman favicon
  - Remove commented code in Gruntfile
  - indent to 4 spaces in test jshintrc
